### PR TITLE
feature/HB-93-Backend-setup-and-configuration-table-review

### PR DESCRIPTION
**Pull request by kahofujita**
#### **Description:**
_Please remember that the reviewer may not be as familiar with work being done or the project as you._

> DESCRIPTIONGOESHERE

**Run command:**

```
npm run
```

- [ ] This code includes new npm packages/libraries

#### **QA instructions (if applicable):** 
1. 
2.

---

- [ ] Unit tests (if needed)
- [ ] Code cleanup (removed dead code, console logs, sorted props, etc)
- [ ] Updated ticket with info and linked to this PR

### CODE REVIEW

File: api/prisma/schema.prisma [hunk 1]
```diff
@@ -9,3 +9,21 @@ datasource db {
   provider = "mysql"
   url      = env("DATABASE_URL")
 }
+
+model Deal {
+  dealId Int @id @default(autoincrement())
+  business @relation(fields: [dealId], references: [businessId])
```
Comment (zyzmoz) [inline api/prisma/schema.prisma:15]: business should be:
`business @relation(fields: [businessId], references: [businessId])`

File: api/prisma/schema.prisma [hunk 2]
```diff
@@ -9,3 +9,21 @@ datasource db {
...
+  service @relation(fields: [dealId], references: [serviceId])
```
Comment (zyzmoz) [inline api/prisma/schema.prisma:16]: service should be:
`service @relation(fields: [serviceID], references: [serviceID])`

File: api/prisma/schema.prisma [hunk 3]
```diff
@@ -9,3 +9,21 @@ datasource db {
...
+  dealAmount Int
+  price Int
+  startDate DateTime
+  endDate DateTime
+}
+
+model Review {
+  reviewId Int @id @default(autoincrement())
+  business @relation(fields: [reviewId], references: [businessId])
```
Comment (zyzmoz) [inline api/prisma/schema.prisma:25]: business should be:
`business @relation(fields: [businessId], references: [businessId])`

File: api/prisma/schema.prisma [hunk 4]
```diff
@@ -9,3 +9,21 @@ datasource db {
...
+  appointment @relation(fields: : [reviewId], references: [appointmentId])
```
Comment (zyzmoz) [inline api/prisma/schema.prisma:26]: appointment should be:
`appointment @relation(fields: : [appointmentId], references: [appointmentId])`

**Review by zyzmoz — CHANGES_REQUESTED** [review]

File: api/prisma/schema.prisma [hunk 5]
```diff
@@ -9,3 +9,21 @@ datasource db {
   provider = "mysql"
   url      = env("DATABASE_URL")
 }
+
+model Deal {
+  dealId Int @id @default(autoincrement())
+  business @relation(fields: [businessID], references: [businessID])
+  service @relation(fields: [serviceID], references: [serviceID])
```
Comment (zyzmoz) [inline api/prisma/schema.prisma:16]: Please add:

`businessID Int`  
`serviceID Int`

File: api/prisma/schema.prisma [hunk 6]
```diff
@@ -9,3 +9,21 @@ datasource db {
...
+  price Int
+  startDate DateTime
+  endDate DateTime
+}
+
+model Review {
+  reviewId Int @id @default(autoincrement())
+  business @relation(fields: [businessID], references: [businessID])
+  appointment @relation(fields: : [appointmentID], references: [appointmentID])
+  appointmentRating Int
```
Comment (zyzmoz) [inline api/prisma/schema.prisma:27]: Please add:

`businessID Int`  
`appointmentID Int`

**Review by zyzmoz — CHANGES_REQUESTED** [review]

**Review by zyzmoz — APPROVED** [review]

**Review by zyzmoz — APPROVED** [review]

**Review by zyzmoz — APPROVED** [review]

**Review by zyzmoz — APPROVED** [review]

**Review by gabrieldiasls — APPROVED** [review]

**Pull request merged**